### ISSUE TITLE
​Mass add tags

### ISSUE THREAD

**Issue by Tealk** [toeverything/AFFiNE#8990]
### Description

Assign tags to all selected documents

### Use case

It is currently possible to import markdown documents in bulk, but then you have to assign tags to them individually. This is very tedious for a changeover.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by CatsJuice** [comment]
@hwangdev97  cc

**Comment by hwangdev97** [comment]
Absolutely, we need more bulk operation features. Adding tags to multiple selections is definitely a viable solution.

**Comment by DrGump** [comment]
Any update on tagging multiple documents at once?

**Comment by sanathusk** [comment]
This is needed features. Can we please reopen the issue. Is the team open to PRs that implement the features?
